tests/postgres_tests/test_functions.py

```diff
@@ -26,3 +27,20 @@ def test_transaction_now(self):
 
         self.assertIsInstance(m1.when, datetime)
         self.assertEqual(m1.when, m2.when)
+
+
+class TestRandomUUID(PostgreSQLTestCase):
+
+    def test_random_uuid(self):
+        m1 = UUIDTestModel.objects.create()
+        m2 = UUIDTestModel.objects.create()
+
+        UUIDTestModel.objects.filter(id=m1.id).update(uuid=RandomUUID())
+        sleep(0.1)
```

Comment:
Why do you need this sleep function? If `RandomUUID` returns the same UUID for two consecutive calls at the same second that seems to be a bug in PostgreSQL.